### ISSUE TITLE
Unable to load the weight from the backup folder

### ISSUE THREAD

**Issue by divy-14** [AlexeyAB/darknet#4770]
I am training my custom model on google colab. When I train the model using darknet detect train after every 100 iterations weights are saved in the backup folder. However, when I try to load these weights it shows the error: Could not load weight from backup/.
Here is an attached screenshot for the same.
![2020-01-29 (2)](https://user-images.githubusercontent.com/43966117/73341656-1228fd00-42a3-11ea-9da6-d1a1258b3b76.png)

**Comment by stephanecharette** [comment]
What exactly is the command you are running, and from which directory are you running the command?

The error you posted is about this file:  `backup/yolo3-tiny-obj_last.weights`

Note the path is relative to the current working directory.

**Comment by divy-14** [comment]
Thank you for the reply,
What I did today was instead of the relative path made the path absolute i.e /content/darknet/backup and it started to work fine. And now it is working fine on images.
However, when I run the darknet detector demo for detecting an object in the video file the command is working fine however it fails to detect any object. To verify that the model was working on the video I took various screenshots from the video and the model was working fine on the images. I tried changing #define NFRAMES 3 to #define NFRAMES 1 as mentioned in some earlier issues but the issue still persists.
